mafContrib/transientAsciiSEDMetric.py

```diff
@@ -117,20 +117,16 @@ def __init__(
         # and thus needs an alternate definition of the output type and units
         # when calling the init of the base class.
         if self.output_data:
-            super(transientAsciiSEDMetric, self).__init__(
-                col=[self.mjdCol, self.m5Col, self.filterCol],
-                metricDtype="object",
-                units="",
-                metricName=metric_name,
-                **kwargs
-            )
+            super_dict = {"units": "", "metricDtype": "object"}
         else:
-            super(transientAsciiSEDMetric, self).__init__(
-                col=[self.mjdCol, self.m5Col, self.filterCol],
-                units="Fraction Detected",
-                metricName=metric_name,
-                **kwargs
-            )
+            super_dict = {"units": "Fraction Detected"}
+
+        super(transientAsciiSEDMetric, self).__init__(
```

Comment:
One more (hopefully helpful) comment .. in python 3, super is improved! We haven't backported this to *all* of our metrics /etc yet, but some have been updated but here's the thing -- you can now just do 
`super().__init__(blahblahblah)` 
instead of having to do 
`super(classname, self).__init__(blahblahblah)`

Why is this a helpful thing to swap to? Mostly because it makes it easier to not make the mistake of changing your metric class name without remembering to change the call line in super. 

Anyway, it's an option if you want to change it.

Reply:
Thanks! I think at this time I'll leave it as is. I am hoping to have this merged soon so others in the DESC Observing Strategy Task Force can use it.